youtube_transcript_api/__main__.py

```diff
@@ -39,18 +39,31 @@ def parse_args(args):
         default=False,
         help='If this flag is set the output will be JSON formatted.',
     )
-
+    parser.add_argument(
+        '--http-proxy', dest='http_proxy',
+        default=None, metavar='URL',
+        help='Use the specified HTTP proxy.'
+    )
+    parser.add_argument(
+        '--https-proxy', dest='https_proxy',
+        default=None, metavar='URL',
+        help='Use the specified HTTPS proxy.'
+    )
     return parser.parse_args(args)
 
 
 def main():
     logging.basicConfig()
 
     parsed_args = parse_args(sys.argv[1:])
+
+    proxies = {"http": parsed_args.http_proxy or "", "https": parsed_args.https_proxy or ""}
```

Comment:
Sorry, I just noticed this. But proxies has to be `None` if neither `parsed_args.http_proxy` nor `parsed_args.https_proxy` are set. Otherwise the `proxies` dict with both empty strings will be used.

Reply:
```suggestion
    if parsed_args.http_proxy or parsed_args.https_proxy:
        proxies = {"http": parsed_args.http_proxy or "", "https": parsed_args.https_proxy or ""}
    else:
        proxies = None
```